search-improvements.rst

```diff
@@ -0,0 +1,143 @@
+.. post:: August 14, 2018
+   :tags: gsoc, search, feature
+   :author: Safwan
+   :location: DHA
+
+Improved Search
+==================================
+Have you ever used the search functionality inside documentations hosted by Read The Docs?
+Mostly yes. Search can always be improved and in the era of Search Engines, the
+documentation search also plays a big role for the users. We have realized the importance,
+and got our GSoC student interested to take the challenge. The main goal was to refactor the
```

Comment:
I think it's ok to write this in first person (I, me, etc.). Introduce yourself! You are doing great work and you get to take credit for it.

Reply:
I see that you introduce yourself below. That's good but I still think you can write in first person.